app/code/Magento/Elasticsearch/SearchAdapter/Query/Builder/Sort.php

```diff
@@ -54,17 +62,20 @@ class Sort
     /**
      * @param AttributeProvider $attributeAdapterProvider
      * @param FieldNameResolver $fieldNameResolver
+     * @param ScopeConfigInterface $scopeConfig
      * @param array $skippedFields
      * @param array $map
      */
     public function __construct(
         AttributeProvider $attributeAdapterProvider,
         FieldNameResolver $fieldNameResolver,
+        ScopeConfigInterface $scopeConfig,
```

Comment:
`ScopeConfigInterface $scopeConfig` should be moved to the end of the contruct because of Backward compatible.
```php 
ScopeConfigInterface  $scopeConfig = null
```

and:

```php
$this->scopeConfig = $scopeConfig ?: ObjectManager::getInstance()->get(ScopeConfigInterface::class);
```